src/plotApplications/components/PlotApplicationsListPage.js

```diff
@@ -441,20 +429,15 @@ class PlotApplicationsListPage extends PureComponent<Props, State> {
     const isTable = visualizationType === VisualizationTypes.TABLE;
     const isMap = visualizationType === VisualizationTypes.MAP;
 
-    const plotApplicationStateFilterOptions = getFieldOptions(plotApplicationsAttributes, 'state', false);
-    const filteredApplications = selectedStates.length
-      ? (applications.filter((application) => selectedStates.indexOf(application.state) !== -1))
-      : applications;
-
-    const count = isMap ? getApiResponseCount(plotApplicationsMapData) : filteredApplications.length;
+    const count = getApiResponseCount(isMap ? plotApplicationsMapData : plotApplicationsListData);
     const columns = this.getColumns();
 
     let amountText;
     if (isMap && isFetchingByBBox || isTable && isFetching) {
       amountText = 'Ladataan...';
     } else {
       if (isMap && Number(getUrlParams(search)?.zoom || 0) < MAX_ZOOM_LEVEL_TO_FETCH_LEASES) {
-        amountText = '';
+        amountText = <>&nbsp;</>;
```

Comment:
Oh, this is why the prop type is ReactNode. Did the ' ' -string break something?

Reply:
A whitespace-only string gets condensed to nothing in HTML (without a non-default `white-space` CSS rule anyway) which meant the text element became 0 pixels tall in this case, thus leading to the inconsistent map positioning in the map view depending on the zoom level.

Reply:
Ahh, the same thing that with line-breaks. I got it. :+1: